crates/codegen/parser/src/chumsky/character_filter.rs

```diff
@@ -109,37 +108,17 @@ impl<'context> CharacterFilter<'context> {
         }
     }
 
-    pub fn default_name(&self) -> Option<String> {
-        if let Self::Negation { child } = self {
-            child.default_name().map(|n| format!("Not{}", n))
-        } else if let Self::Char { char, .. } = self {
-            Some(naming::name_of_terminal_char(*char))
-        } else {
-            None
-        }
-    }
-
-    pub(super) fn to_code(
-        &self,
-        name: Option<&String>,
-        code: &mut CodeGenerator,
-        macro_prefix: &str,
-    ) -> TokenStream {
-        let macro_name = format_ident!("{}terminal", macro_prefix);
-        if let CharacterFilter::Char { char } = self {
-            let kind = code.add_terminal_kind(char.to_string());
-            quote!(#macro_name!(#kind, #char))
-        } else {
-            let predicate = self.to_predicate(false);
-            if let Some(kind) = name.map(|n| code.add_token_kind(n.clone())) {
-                quote!(#macro_name!(#kind, |&c: &char| #predicate))
-            } else {
-                quote!(#macro_name!(|&c: &char| #predicate))
-            }
-        }
-    }
-
-    fn to_predicate(&self, negated: bool) -> TokenStream {
+    // pub fn default_name(&self) -> Option<String> {
```

Comment:
Thoughts on removing unused code like this (instead of commenting it out?).

It is never lost anyways, since we can fetch it from branch history, but helps keep `HEAD` healthy, and cheaper to maintain.

Reply:
Agreed